llvm/tools/repo2obj/R2OELFOutputSection.h

```diff
@@ -166,8 +166,8 @@ template <typename ELFT> class OutputSection {
   llvm::Optional<std::vector<uint8_t>> const &Prefix_;
   GroupInfo<ELFT> *Group_ = nullptr;
 
-  static constexpr auto UnknownIndex = std::numeric_limits<unsigned>::max();
-  unsigned Index_ = UnknownIndex; // The section header table index
+  static constexpr auto UnknownIndex = std::numeric_limits<size_t>::max();
+  size_t Index_ = UnknownIndex; // The section header table index
```

Comment:
Why is `size_t` type not `uint32_t` type?

Reply:
Because the value holds the `size_t` result of calling `State.SectionHeaders.size()` (see repo2obj.cpp line 590). At some point this will be truncated to 32 bits for the ELF file, but I want to keep the internal state consistent.